src/libraries/System.Text.RegularExpressions/gen/DiagnosticDescriptors.cs

```diff
@@ -0,0 +1,85 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using Microsoft.CodeAnalysis;
+
+namespace System.Text.RegularExpressions.Generator
+{
+    public static class DiagnosticDescriptors
```

Comment:
I assume all these diagnostics shouldn't be suppressible. If that's the case, the descriptor should have custom tags containing [`WellKnownDiagnosticTags.NotConfigurable](https://docs.microsoft.com/en-us/dotnet/api/microsoft.codeanalysis.wellknowndiagnostictags.notconfigurable?view=roslyn-dotnet-3.11.0)

Also was this intended to be public?